src/pages/SearchPage.js

```diff
@@ -32,82 +31,73 @@ const propTypes = {
 
     /** All reports shared with the user */
     reports: PropTypes.objectOf(reportPropTypes),
-
-    /** Window Dimensions Props */
-    ...windowDimensionsPropTypes,
-
-    ...withLocalizePropTypes,
 };
 
-const defaultProps = {
-    betas: [],
-    personalDetails: {},
-    reports: {},
-};
+function SearchPage({betas = [], personalDetails = {}, reports= {}}) {
```

Comment:
```suggestion
function SearchPage({betas = [], personalDetails = {}, reports = {}}) {
```

Super nitpick sorry XD